components/sponsored_benefits/app/controllers/sponsored_benefits/organizations/plan_design_proposals_controller.rb

```diff
@@ -4,12 +4,24 @@ class Organizations::PlanDesignProposalsController < ApplicationController
     include DataTablesAdapter
 
     before_action :load_plan_design_organization
-    before_action :load_plan_design_proposal, only: [:edit, :update, :destroy]
+    before_action :load_plan_design_proposal, only: [:edit, :update, :destroy, :publish_quote]
 
     def index
       @datatable = effective_datatable
     end
 
+    def publish_quote
+      #FIXME, any specific conditions to be verified before publishing?
```

Comment:
I think to publish all we need is a plan_design_organization with a primary office location (require zip / county) and a sic_code.

The requirements become a bit stricter when the employer CLAIMS the quote (we need to ensure that the employer has the same zip/county)